client/gui_rpc_server_ops.cpp

```diff
@@ -1195,7 +1195,7 @@ static void handle_get_app_config(GUI_RPC_CONN& grc) {
         return;
     }
     sprintf(path, "%s/%s", p->project_dir(), APP_CONFIG_FILE_NAME);
-    printf("path: %s\n", path);
+    // printf("path: %s\n", path);    // debug only
```

Comment:
Could be completely removed

Reply:
Indeed it could, but I don't like deleting other people's code until they've had a chance to consider why they wrote it (and left it in).

24 hours without comment, and it'll be gone.

Reply:
It's definitely a debug left-over, and we can wait for 24 hours :)

Reply:
I agree with removing the line, git will remember the change

Reply:
Line removed as requested.